learn/nginx.conf

```diff
@@ -0,0 +1,66 @@
+user user user;
+worker_processes 1;
+error_log /var/log/nginx/error.log;
```

Comment:
Could this just say /dev/stderr or similar directly, instead of jumping through the symlink hoop?

Reply:
this can say `stderr`